README.md

```diff
@@ -222,6 +222,11 @@ Now follow the steps to "[finish the installation](#finish-the-installation)".
 
 ## Change Log
 
+### New in 2.64.11
```

Comment:
Please add that `pyenv rehash` is needed after upgrade.

Reply:
Like this? or before the list?